T1S-FQ,T1S-FA,T1S-FA_LLM,T1S-FA_LLM_R,T1S-FA_CIMBUR,T1S-FA_CIMBUR_R,T1S-FA_CFG,T1S-FA_CFG_R,T1S-FA_Query,T1S-FA_Query_R,T1S-FA_MT,T1S-FA_MT_R,T1S-FA_COVE,T1S-FA_COVE_R,T1S-FA_COVE_MT,T1S-FA_COVE_MT_R
Where in the code exception of issue 18151 occur?,The exception of issue 18151 occurs from JvmErgonomics.java file at line 98,The exception of issue 18151 occurs in the body of the issue.,0,The exception in issue 18151 can be triggered at line 98 in the JvmErgonomics.java file in the Elasticsearch server.,1,The exception of issue 18151 occurs in the code at line 98 of the JvmErgonomics.java file in the Elasticsearch server.,1,"The code snippet where the exception mentioned in issue 18151 is triggered is as follows:

```
Description:
Investigating to run elasticsearch with Semeru runtimes reveals that few OpenJDK options is essential to use the application.
The initial results and experiments are on ppc64le, but it should not differ for other platforms running Semeru.

Elasticsearch is very tightly...
```",0,The exception for issue 18151 occurs in the body of the code related to running Elasticsearch with Semeru runtimes.,0,The exception of issue 18151 occurs in the body of the code.,0,The exception for issue 18151 occurs in the body of the code.,0
Find the vmstate from the diagnostic file attached in issue 17458,vmState=0x00000000,"The vmstate is ""openj9-0.36.0"".",0,"The vmstate information from the diagnostic file attached in issue 17458 is ""Java -version output\r\n--------------------\r\n\r\n> jdk8u362-b09_openj9/bin/java -version\r\nopenjdk version ""1.8.0_362""\r\nIBM Semeru Runtime Open Edition (build 1.8.0_362-b09)\r\nEclipse OpenJ9 VM (build openj9-0.36.0, JRE 1.8.0 Linux amd64-64-Bit Compressed References 20230206_588 (JIT enabled, AOT..."". The Java version from the vmstate information in the diagnostic file attached in issue 17458 is ""1.8.0_362"". The build of the OpenJ9 VM from the vm is 17.0.4.1-ea, 17.0.6, 17.0.7+5. The attached diagnostic file for issue 17458 contains vmstate information for Java 8, build 1.8.0_362-b09, using the Eclipse OpenJ9 VM. The vmstate data included in the diagnostic file attached to issue 17458 is the output of",0,"The vmstate is ""Eclipse OpenJ9 VM (build openj9-0.36.0, JRE 1.8.0 Linux amd64-64-Bit Compressed References 20230206_588 (JIT enabled, AOT..."".",0,"Java -version output
--------------------

> jdk8u362-b09_openj9/bin/java -version
openjdk version ""1.8.0_362""
IBM Semeru Runtime Open Edition (build 1.8.0_362-b09)
Eclipse OpenJ9 VM (build openj9-0.36.0, JRE 1.8.0 Linux amd64-64-Bit Compressed References 20230206_588 (JIT enabled, AOT...",0,"The vmstate in the diagnostic file attached to issue 17458 is ""Java -version output"". It can be found in the body of the issue.",0,"Java -version output
jdk8u362-b09_openj9/bin/java -version
openjdk version ""1.8.0_362""
IBM Semeru Runtime Open Edition (build 1.8.0_362-b09)
Eclipse OpenJ9 VM (build openj9-0.36.0, JRE 1.8.0 Linux amd64-64-Bit Compressed References 20230206_588 (JIT enabled, AOT...",0,"The vmstate in the diagnostic file attached to issue 17458 is ""Java -version output"". It can be found in the body of the issue.",0
Which causes the exception in issue18400?,java.lang.AssertionError,The exception in issue18400 is caused by OpenJ9 specific CI failures.,0,The cause of the exception in issue 18400 is a java.lang.AssertionError caused by a failure in the OpenJ9 JIT compiler. This issue is related to the JIT compiler and was raised by a user in the MH project.,1,The exception caused by issue18400 is a java.lang.AssertionError.,1,"Hello from Apache Lucene project, where we run CI builds with various JVMs and versions, including OpenJ9.

We sometimes see CI failures that seem to be OpenJ9 specific, and we don't have many dev resources to pursue them.  You can see the recent failures in Apache mail list...",0,The exception in issue18400 is caused by CI failures specific to OpenJ9 in the Apache Lucene project.,0,The exception in issue18400 is caused by 'Apache Lucene CI builds sometimes fail with OpenJ9 specific issues'.,0,The exception in issue18400 is caused by CI failures specific to OpenJ9 in the Apache Lucene project.,0
Under what condition does the error in issue 17265 occur?,The error in issue 17265 occurs when a call is made to a REST endpoint after class retransformation by a Java agent.,The error in issue 17265 occurs when a call is made after class retransformation by a Java agent in a Spring Boot application deployed in WebSphere Liberty.,1,"The error in issue 17265 was caused by a AnnotationFormatError with the message ""java.lang.IllegalArgumentException: Wrong type at constant pool index"". It occurred while hitting a REST Endpoint in a Spring Boot application deployed in WebSphere Liberty, and only occurred when the call was made after class retransformation by a Java agent.",1,The error in issue 17265 occurs when a call is made after class retransformation by a Java agent.,1,The error in issue 17265 occurs when hitting a REST Endpoint in a Spring Boot application deployed in WebSphere Liberty after class retransformation by a Java agent. The error is AnnotationFormatError: java.lang.IllegalArgumentException: Wrong type at constant pool index. This issue was raised by the user and is related to the VM component.,1,The error in issue 17265 occurs when hitting a REST Endpoint in a Spring Boot application deployed in WebSphere Liberty after class retransformation by a Java agent.,1,"The error in issue 17265, ""AnnotationFormatError: java.lang.IllegalArgumentException: Wrong type at constant pool index post class retransformation by agent"", occurs when hitting a REST Endpoint in a Spring Boot application deployed in WebSphere Liberty. It only occurs when the call was made after class retransformation by a Java agent.",0,The error in issue 17265 is triggered by hitting a REST Endpoint in a Spring Boot application deployed in WebSphere Liberty after class retransformation by a Java agent.,1
which java version is used in issue 18845?,21.0.2,"Java version ""21.0.2"" is used in issue 18845.",1,"
The specified Java version for issue 18845 is ""21.0.2"".",1,"Java version ""21.0.2"" is used in issue 18845.",1,"Java version specified in issue 18845 is ""21.0.2""",1,"Java version ""21.0.2"" is used in issue 18845.",1,"Java version ""21.0.2"" is specifically mentioned in the details of issue 18845.",1,"Java version being used in issue 18845 is ""21.0.2""",1
What exception occured in the issue 17120,The exception that occurred in issue 17120 is IllegalMonitorStateException or java.lang.IllegalMonitorStateException,ArrayIndexOutOfBoundsException,0,"The exception that occurred in issue 17120 is an IllegalMonitorStateException, with a count of 29106.",1,"Exception in thread """" java.lang.IllegalMonitorStateException\nat java.base/java.util.concurrent.locks.ReentrantLock$Sync.tryRelease(ReentrantLock.java:175)\nat java.base/jdk.internal.vm.Continuation.enter(Continuation.java:171)\n\n java.lang.RuntimeException: count = 29106\nat...",0,"The exception that occurred in issue 17120 was ""jdk20 OpenJDK Windows virtual/stress/PinALot.java IllegalMonitorStateException"".",1,IllegalMonitorStateException,1,"IllegalMonitorStateException was found in the issue report with number 17120 titled ""jdk20 OpenJDK Windows virtual/stress/PinALot.java IllegalMonitorStateException"".",1,IllegalMonitorStateException was encountered in issue 17120.,1
How many times grinder reproduced the the issue 18102?,2,0,0,1,0,1,0,The grinder reproduced the issue in issue 18102 10 times.,0,1 instance of the issue 18102 was reproduced by the grinder.,0,The grinder reproduced the issue 18102 10 times.,0,1 instance of the issue 18102 was reproduced by the grinder.,0
which class throw ArrayIndexOutOfBoundsException in issue 19014?,TestArrayCopy,java/lang/String/ToLowerCase,0,The class that throws ArrayIndexOutOfBoundsException in issue 19014 is TestArrayCopy.,1,java.lang.ArrayIndexOutOfBoundsException,0,The class that throws ArrayIndexOutOfBoundsException in issue 19014 is not explicitly mentioned in the provided data.,0,"The class responsible for throwing the ArrayIndexOutOfBoundsException in issue 19014 is ""java/lang/String/ToLowerCase"".",0,"The class ""OpenJ9-11 Incorrect array optimizations"" can be found in the codebase and it is associated with the issue report number 19014 that mentions ArrayIndexOutOfBoundsException.",0,"ArrayIndexOutOfBoundsException can be thrown in issue 19014, but the specific class responsible for throwing it is not explicitly mentioned in the provided information.",0
Which IBM Semeru Edition reported in issue 17019?,17.0.6.0,IBM Semeru Runtime Open Edition 17.0.6.0,1,"Yes, 17.0.6.0 is the version of IBM Semeru Edition mentioned in issue 17019.",1,IBM Semeru Runtimes,0,IBM Semeru Edition mentioned in issue 17019 is version 17.0.6.0.,1,"IBM Semeru Edition reported in issue 17019 is ""IBM Semeru Runtime Open Edition 17.0.6.0"".",1,IBM Semeru Runtime Open Edition 17.0.6.,0,"IBM Semeru Edition reported in issue 17019 is ""Open Edition 17.0.6.0"".",1
What triggered the exception in issue 18082?,The exception in issue 18082 is caused by ArrayIndexOutOfBoundsException or java.lang.ArrayIndexOutOfBoundsException in the aarch64_linux build,"The failure was triggered by an internal build of OpenJDK version ""21-internal"" 2023-09-19 on aarch64 Linux.",0,"The exception in issue 18082 is caused by an ArrayIndexOutOfBoundsException in the aarch64_linux build test ant task. This is indicated by the title, body, exception_and_failure_stack_trace, and labels of the issue.",1,The exception triggered in issue 18082 was a java.lang.ArrayIndexOutOfBoundsException.,1,"The cause of the exception in issue 18082 is that the ""aarch64_linux build test ant task failed with ArrayIndexOutOfBoundsException in some machines"". This is based on the title, body, exceptions, and labels provided in the issue.",1,"The exception in issue 18082 was triggered by a failure in the internal build with OpenJDK version ""21-internal"" on 2023-09-19.",0,"The exception in issue 18082 was triggered by an internal build with OpenJDK version '21-internal' on September 19, 2023.",0,"The exception in issue 18082 was caused by an internal build with OpenJDK version ""21-internal"" from 2023-09-19.",0
Which error occurs in SpinedBufferTest of issue 17774?,NullPointerExceptions,NullPointerExceptions are seen in SpinedBufferTest.,1,NullPointerExceptions are seen in SpinedBufferTest. This failure was first noticed in the JDK11 Linux aarch64 sanity.openjdk PR build for #17732. This failure is highly intermittent; it happens once in 200 attempts. The error is a NullPointerException in the SpinedBufferTest class in the java.util.stream package.,1,The error that occurs in SpinedBufferTest of issue 17774 is a java.lang.NullPointerException.,0,"NullPointerExceptions are seen in SpinedBufferTest. This failure was first noticed in the JDK11 Linux aarch64 sanity.openjdk PR build for issue 17732. The failure is highly intermittent, happening once in 200.",1,"The error that occurs in SpinedBufferTest of issue 17774 is ""NullPointerExceptions seen in SpinedBufferTest"".",1,"The error that occurs in SpinedBufferTest of issue 17774 is ""NullPointerExceptions"".",1,"The error that occurs in SpinedBufferTest of issue 17774 is ""NullPointerExceptions"".",1
What is the specific failure reported in the test testGetAttributes in 18081?,java.lang.AssertionError,The specific failure reported in the test testGetAttributes in 18081 is that there is no z/OS machines available on OpenJ9 jenkins for testing.,0,The specific failure reported in the testGetAttributes test in issue 18081 is that there is no public link available for testing on z/OS machines on OpenJ9 Jenkins. The sub-test that fails is the functional testList.,0,The specific failure reported in the test testGetAttributes in 18081 is a java.lang.AssertionError.,1,"Failure in JLM_Tests_interface_0/1 on z/OS 3.1. There isn't a public link as there is no z/OS machines available on OpenJ9 jenkins for testing. Specifically this is the sub-test that fails. This is a brand new system setup undergoing validation, but I think this is a real failure... BUILDLIST=functional TARGET=`testList...",0,"The specific failure reported in the test testGetAttributes in issue number 18081 is: ""There isn't a public link as there is no z/OS machines available on OpenJ9 jenkins for testing. Specifically this is the sub-test that fails. This is a brand new system setup undergoing validation, but I think this is a real failure... BUILDLIST=functional TARGET=testList...""",0,"The specific failure reported in the test testGetAttributes in issue number 18081 is: ""Failure in JLM_Tests_interface_0/1 on z/OS 3.1. The sub-test that specifically fails is the ""functional"" sub-test. The system setup for testing in issue number 18081 is a brand new setup undergoing validation, and the failure reported is considered a real failure.""",0,The specific failure reported in the test testGetAttributes in issue number 18081 is related to the sub-test Failure in JLM_Tests_interface_0/1 on z/OS 3.1.,0
What specific exception is raised in issue 18043?,java.lang.Exception: ERROR: Ciphertext does not match,The specific exception raised in issue 18043 is an ArrayIndexOutOfBoundsException.,0,"The exception raised in issue 18043 is a java.lang.Exception with the message ""ERROR: Ciphertext does not match"".",1,java.lang.Exception: ERROR: Ciphertext does not match,1,The exception raised in issue 18043 is related to the test failure in the FIPS JDK17 environment due to a mismatch in the ciphertext.,0,"The exception specifically raised in issue 18043 is ""Ciphertext does not match"".",1,"The specific exception raised in issue 18043 is ""Ciphertext does not match"".",1,"The specific exception raised in issue 18043 is ""Ciphertext does not match"".",1
What component is affected by issue 18718?,comp:vm,"The component affected by issue 18718 is ""comp:vm"".",1,"The affected component in issue 18718 is ""comp:vm"", ""test failure"", and ""jdk22"" based on its description and labels.",1,"The component affected by issue 18718 is ""comp:vm"".",1,"The affected component in issue 18718 is ""comp:vm"".",1,"The component affected by issue 18718 is ""comp:vm, test failure, jdk22"".",1,"The component affected by issue 18718 is ""comp:vm"".",1,"The component affected by issue 18718 is ""comp:vm"".",1
Show the stack trace of issue 17843,"Exception in thread ""Thread-0"" org.eclipse.openj9.criu.SystemCheckpointException: Could not dump the JVM processes, err=-52
at openj9.criu/org.eclipse.openj9.criu.CRIUSupport.checkpointJVMImpl(Native Method)
at openj9.criu/org.eclipse.openj9.criu.CRIUSupport.checkpointJVM(CRIUSupport.java:677)
at org.openj9.criu.CRIUTestUtils.checkPointJVMNoSetup(CRIUTestUtils.java:105)
at org.openj9.criu.TestSingleThreadModeRestoreException$2.run(TestSingleThreadModeRestoreException.java:123)
at java.base/java.lang.Thread.run(Thread.java:857)","Failure link
------------

From [an internal build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk17_j9_sanity.functional_ppc64le_linux_testList_0/414/)(`rhel7lert-1-9`):
```
java version ""17.0.8"" 2023-07-18
IBM Semeru Runtime Certified Edition 17.0.8.0-rc1 (build...",0,"The stack trace details for issue 17843 are: Exception in thread ""Thread-0"" org.eclipse.openj9.criu.SystemCheckpointException: Could not dump the JVM processes, err=-52
at openj9.criu/org.eclipse.openj9.criu.CRIUSupport.checkpointJVMImpl(Native Method)
at java.base/java.lang.Thread.run(Thread.java:857)",1,"Exception in thread ""Thread-0"" org.eclipse.openj9.criu.SystemCheckpointException: Could not dump the JVM processes, err=-52
at openj9.criu/org.eclipse.openj9.criu.CRIUSupport.checkpointJVMImpl(Native Method)
at java.base/java.lang.Thread.run(Thread.java:857)",1,"The stack trace details of issue 17843 are: 

Title: CRIU Assertion failed at openj9/runtime/compiler/control/CompilationThread.cpp:2927: !isCheckpointInProgress() Checkpoint already in progress!
Body: Failure link
------------
From an internal build: [an internal build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk17_j9_sanity.functional_ppc64le_linux_testList_0/414/)(`rhel7lert-1-9`):
```
java version ""17.0.8"" 2023-07-18
IBM Semeru Runtime Certified Edition 17.0.8.0-rc1 (build...)",0,"Failure link
------------
From [an internal build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk17_j9_sanity.functional_ppc64le_linux_testList_0/414/)(`rhel7lert-1-9`):
```
java version ""17.0.8"" 2023-07-18
IBM Semeru Runtime Certified Edition 17.0.8.0-rc1 (build...",0,The stack trace of issue 17843 can be found in the build logs of the internal build with java version 17.0.8.,0,The stack trace for issue 17843 can be found in the failure link provided.,0
Which file was not found during the test execution in issue #17540?,testng_210.xml,The file that was not found during the test execution in issue #17540 was not specified in the body of the issue.,0,java.io.FileNotFoundException: testng_210.xml (No such file or directory),1,The file that was not found during the test execution in issue #17540 was testng_210.xml.,1,The file that was not found during the test execution in issue #17540 is located at the following link: https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdkNext_j9_sanity.functional_x86-64_linux_Personal/422/tapResults/,0,The file that was not found during the test execution in issue #17540 is not explicitly mentioned in the body of the issue.,0,"The file that was not found during the test execution in issue #17540 is testng_210.xml, as mentioned in the title and body of the issue. Additional details can be found at the provided link: https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdkNext_j9_sanity.functional_x86-64_linux_Personal/422/tapResults/",1,"The file that was not found during the test execution in issue #17540 is ""testng_210.xml"".",1
Which line of code in the CharacterDataLatin1 class triggered the ArrayIndexOutOfBoundsException in issue 17063?,72,"The line of code in the CharacterDataLatin1 class that triggered the ArrayIndexOutOfBoundsException in issue 17063 is ""java/lang/String/ToLowerCase.java"".",0,The specific line of code that caused the ArrayIndexOutOfBoundsException in issue 17063 is line 72 in the CharacterDataLatin1 class.,1,The line of code that triggered the ArrayIndexOutOfBoundsException in issue 17063 is line 72 in the CharacterDataLatin1 class.,1,"The specific line of code in the CharacterDataLatin1 class that caused the ArrayIndexOutOfBoundsException in issue 17063 is in the ""java/lang/String/ToLowerCase.java"" file.",0,"The specific line of code in the CharacterDataLatin1 class that caused the ArrayIndexOutOfBoundsException in issue 17063 is in the ""java/lang/String/ToLowerCase.java"" file.",0,"The specific line of code in the ""java/lang/String/ToLowerCase.java"" file that caused the ArrayIndexOutOfBoundsException in issue 17063 is not provided in the body of the issue.",0,"The ArrayIndexOutOfBoundsException in issue 17063 was triggered by the line of code in the CharacterDataLatin1 class that is in the ""java/lang/String/ToLowerCase.java"" file.",0
What were the configurations of the last passing build in issue 17359?,"Java version: ""21-internal"" dated 2023-09-19
OpenJDK Runtime Environment: 21-internal-adhoc.jenkins.BuildJDKnextx86-64linuxvalhallaNightly
Eclipse OpenJ9 VM: Build master-0609eb43a77, JRE 21 Linux amd64-64-Bit Compressed References 20230503_1032 (JIT enabled, AOT enabled)
OpenJ9 version: 0609eb43a77
OMR version: 5dcc2f28955
JCL version: 004eb7f92c8 based on jdk-21+19","The last passing build in issue 17359 had the following configurations: Failure link, regression introduced: yes, Failed in internal nightly [build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdkValhalla_j9_sanity.functional_x86-64_linux_valhalla_Nightly/856/tapResults/), java -version, openjdk version ""21-internal"".",1,"The most recent successful build in issue 17359 had a failure link and failed in an internal nightly build. The java version used was openjdk version ""21-internal"".",0,The last passing build in issue 17359 had the following configurations: [Internal build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk8_j9_sanity.openjdk_s390x_linux_openj9_jdk8_2023-03-29-22-01-01/).,1,"Failure link
------------
regression introduced: yes 
Failed in internal nightly [build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdkValhalla_j9_sanity.functional_x86-64_linux_valhalla_Nightly/856/tapResults/):
```
java -version
openjdk version ""21-internal""...",1,"The configurations of the last passing build in issue 17359 were:
Java version ""17.0.6""
IBM Semeru Runtime Open Edition 17.0.6.0
Eclipse OpenJ9 VM 17.0.6.0",0,"The configurations of the last passing build in issue 17359 did not include Java version '17.0.6', IBM Semeru Runtime Open Edition 17.0.6.0 (build 17.0.6+10), or Eclipse OpenJ9 VM 17.0.6.0 (build openj9-0.36.0, JRE 17 Linux amd64-64-Bit Compressed References 20230117_397 (JIT enabled, AOT enabled).",0,"The configurations for the last passing build in issue 17359 are as follows:
- Title: testGetSizeOfArray failed in Valhalla builds
- State: closed
- Comments: 7
- Created at: 2023-05-08
- Closed at: 2023-05-15
- Issue creator: hangshao0
- Labels: test failure
- Body: Failure link
  regression introduced: yes 
  Failed in internal nightly build: [link](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdkValhalla_j9_sanity.functional_x86-64_linux_valhalla_Nightly/856/tapResults/)
- Milestone: None",0
Write the parameter mentioned in StackWalkerTest in issue 18961?,-Xjit:count=0,The parameter mentioned in StackWalkerTest in issue 18961 is `-Xjit:count=0`.,1,java.lang.AssertionError: caller class wrong expected [class org.openj9.test.stackWalker.StackWalkerTest] but found [class java.lang.StackWalker]\nat org.testng.Assert.fail(Assert.java:96)\nat org.openj9.test.stackWalker.StackWalkerTest.testGetCallerClass(StackWalkerTest.java:92),0,The parameter mentioned in StackWalkerTest in issue 18961 is `-Xjit:count=0`.,1,"https://openj9-jenkins.osuosl.org/job/Pipeline-Build-Test-JDK11/742/ - all tested platforms

https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_extended.functional_aarch64_linux_Nightly_testList_0/604

StackWalkerTest_1 `-Xjit:count=0`

```
23:24:51  FAILED: testGetCallerClass
23:24:51 ...",0,The parameter mentioned in StackWalkerTest for issue 18961 is `-Xjit:count=0`.,1,The parameter mentioned in StackWalkerTest in issue 18961 is `-Xjit:count=0`.,1,The parameter mentioned in StackWalkerTest for issue 18961 is `-Xjit:count=0`.,1
What JVM option was seen in the test failure of issue 18447?,-XX:+UseCompressedOops.,"The JVM option seen in the test failure of issue 18447 is ""java version"".",0,"The JVM option associated with the test failure in issue 18447 is ""-XX:+UseAOT"".",0,"The JVM option seen in the test failure of issue 18447 is ""java version"".",0,"The JVM option mentioned in the test failure of issue 18447 is ""17.0.10-beta"".",0,"The JVM option observed in the test failure related to issue 18447 is ""java version ""17.0.10-beta"" 2024-01-16 IBM Semeru Runtime Certified Edition 17.0.10+2-202311111534"".",0,"The JVM option seen in the test failure of issue 18447 was ""java version ""17.0.10-beta"" 2024-01-16"".",0,"The JVM option observed in the test failure related to issue 18447 is ""17.0.10-beta"".",0
What was the failure reported in issue 18277?,"java.security.ProviderException: Secmod directory /etc/pki/nssdb invalid, NSS already initialized ",The failure reported in issue 18277 was an ArrayIndexOutOfBoundsException in the java/lang/String/ToLowerCase test.,0,"The failure reported in issue 18277 was an internal build failure with the following details: 
-Dsemeru.fips=true
Failure link
From an internal build (https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk11_j9_extended.openjdk_x86-64_linux_fips140_2_testList_2/3/)(rhel8x64-fips4-1):
openjdk version ""11.0.21"" 2023-10-17
IBM Semeru Runtime Open Edition...",0,"The failure reported in issue 18277 was a java.security.ProviderException with the message ""Secmod directory /etc/pki/nssdb invalid, NSS already initialized with..."".",1,"The failure reported in issue 18277 is related to a test failure with the label ""comp:fips"". The description includes information about the failure and a link to an internal build.",0,The failure reported in issue 18277 was related to a FIPS provider exception with a specific directory being invalid.,0,"The failure reported in issue 18277 was associated with the specific error message ""-Dsemeru.fips=true\r\n\r\nFailure link\r\n------------\r\n\r\nFrom [an internal build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk11_j9_extended.openjdk_x86-64_linux_fips140_2_testList_2/3/)(`rhel8x64-fips4-1`):\r\n```\r\nopenjdk version ""11.0.21"" 2023-10-17\r\nIBM Semeru Runtime Open Edition..."".",0,The failure reported in issue 18277 was related to a FIPS ProviderException with the Secmod directory being invalid. The known solution or workaround for this issue is to use the command line argument: -Dsemeru.fips=true.,0
On which machine did the failure occur for issue 17137?,rhel9lert-2,"The failure occurred on machine ""rhel9lert-2"".",1,The machine name where the failure occurred for issue 17137 is rhel9lert-2.,1,The failure occurred on the machine with the java.security.ProviderException.,0,The machine name where the failure occurred for issue 17137 is 'rhel9lert-2'.,1,The failure for issue 17137 occurred on machine 'rhel9lert-2'.,1,The failure for issue 17137 did not occur only on machine 'rhel9lert-2'.,1,The machine affected by the failure for issue 17137 was 'rhel9lert-2'.,1
what is the nature of the failure in issue 17995?,The failure in issue 17995 is caused by java.lang.ArrayIndexOutOfBoundsException or ArrayIndexOutOfBoundsException,The nature of the failure in issue 17995 is a failure in an internal build.,0,"The failure in issue 17995 is a test failure in the JDK NIO library, specifically in the TestIBMBugs.java file. The failure is caused by an ArrayIndexOutOfBoundsException in the CharacterDataLatin1 class. This issue has the label ""test failure"" and the body includes a link to an internal build that shows the failure.",0,The nature of the failure in issue 17995 is an ArrayIndexOutOfBoundsException.,1,"Based on the title, exceptions, body, and labels of issue 17995, the nature of the failure is related to a test failure in the TestIBMBugs.java file, specifically encountering a java.lang.ArrayIndexOutOfBoundsException at java.lang.CharacterDataLatin1.getProperties(CharacterDataLatin1.java:72). The failure occurred in an internal build of openjdk version ""1.8.0_392-beta"" of IBM Semeru Runtime Open Edition. The issue is labeled as a test failure.",0,The nature of the failure in issue 17995 is a java.lang.ArrayIndexOutOfBoundsException in the file TestIBMBugs.java.,1,The nature of the failure in issue 17995 is a java.lang.ArrayIndexOutOfBoundsException.,1,"The nature of the failure in issue 17995 is a java.lang.ArrayIndexOutOfBoundsException caused by an internal build with OpenJDK version ""1.8.0_392-beta"" and IBM Semeru Runtime Open Edition.",1
in which method the exception occurs in issue 18487,ElapsedTime.main,"The exception occurs in the body of the issue, specifically in the ""Failure link"" section.",0,The exception is happening in the ElapsedTime.main method.,1,"The exception occurs in the ""Failure link"" method.",0,The exception occurs in the body of the issue.,0,"The exception occurs in the method ""cmdLineTest_J9test_elapsedTime_0"" in issue 18487.",0,The exception occurs in the method mentioned in the body of the issue.,0,The exception occurs in the method described in the body of the issue 18487.,0
Which test failed in issue 18694?,test_asInterfaceInstance_InnerClass,The test that failed in issue 18694 is jsr292Test.,0,The failing test in issue 18694 is the jsr292Test.,0,The test that failed in issue 18694 is jsr292Test.,0,"The failed test in issue 18694 is related to ""jdk22 jsr292Test NoClassDefFoundError SamePackageSingleMethodInterfaceExample/0x0000000000000000 (wrong name: jdk/MHProxy1/SamePackageSingleMethodInterfaceExample)"". The detailed information about the test failure can be found in the body of the issue.",0,"The test that failed in issue 18694 was ""jsr292Test"".",0,"The test that failed in issue 18694 is not ""jsr292Test"".",0,"The test that failed in issue 18694 was ""jsr292Test"".",0
Which class and method in the codebase does the stack trace point to in issue 18698?,"BasicTest class, testPrimitiveConversion method.","The stack trace points to the class ""java/lang/invoke/MethodHandleProxies/BasicTest.java"" in the codebase.",0,The class and method referenced by the stack trace in issue 18698 is BasicTest.testPrimitiveConversion.,1,The stack trace points to the java/lang/invoke/MethodHandleProxies/BasicTest.java class and the java/util/function/jdk/MHProxy1/Function/0x0000000000000000 method.,0,"The class and method indicated by the stack trace in issue 18698 is ""java/lang/invoke/MethodHandleProxies/BasicTest"".",0,"The stack trace in issue 18698 points to the class and method ""java/lang/invoke/MethodHandleProxies/BasicTest.java"".",0,The stack trace in issue 18698 does not point to the class 'jdk_lang' or the method 'java/lang/invoke/MethodHandleProxies/BasicTest.java'.,0,"The stack trace in issue 18698 is pointing to the class ""java/lang/invoke/MethodHandleProxies/BasicTest.java"" and the method ""jdk_lang"".",0
List some of the heap options Elasticsearch uses with the UseG1GC option in issue 18151,"-XX:G1HeapRegionSize
-XX:InitiatingHeapOccupancyPercent
-XX:G1ReservePercent","Elasticsearch uses the following heap options with the UseG1GC option in issue 18151: -XX:+UseG1GC, -XX:MaxGCPauseMillis=200, -XX:InitiatingHeapOccupancyPercent=30, -XX:G1HeapRegionSize=32M, -XX:ConcGCThreads=4, -XX:ParallelGCThreads=4, -XX:G1ReservePercent=15, -XX:G1HeapWastePercent=5, -XX:G1MixedGCCountTarget=8, -XX:G1OldCSetRegionThresholdPercent=10, -XX:G1MixedGCLiveThresholdPercent=85, -XX:G1RSetUpdatingPauseTimePercent=5, -XX:SurvivorRatio=6, -XX:G1NewSizePercent=20, -XX:G1MaxNewSizePercent=60, -XX:G1HeapRegionSize=32M, -XX:MaxGCPauseMillis=200, -XX:G1HeapWastePercent=5, -XX:G1MixedGCCountTarget=8, -XX:G1OldCSetRegionThresholdPercent=10, -XX:G1",0,"The heap options used by Elasticsearch are ""-Xms1g -Xmx1g -XX:+UseConcMarkSweepGC -XX:CMSInitiatingOccupancyFraction=75 -XX:+UseCMSInitiatingOccupancyOnly -Des.networkaddress.cache.ttl=60 -Des.networkaddress.cache.negative.ttl=10"".",0,"Elasticsearch uses the following heap options with the UseG1GC option in issue 18151: 
-XX:+UseG1GC
-XX:MaxGCPauseMillis=200
-XX:InitiatingHeapOccupancyPercent=30
-XX:G1HeapRegionSize=32m
-XX:MinHeapFreeRatio=5
-XX:MaxHeapFreeRatio=10
-XX:G1NewSizePercent=40
-XX:G1MaxNewSizePercent=50
-XX:G1ReservePercent=15
-XX:G1HeapWastePercent=5
-XX:G1MixedGCCountTarget=8
-XX:G1OldCSetRegionThresholdPercent=10
-XX:G1MixedGCLiveThresholdPercent=85
-XX:G1RSetUpdatingPauseTimePercent=5
-XX:SurvivorRatio=6
-XX:G1MixedGCCountTarget=8
-XX:G1OldCSetRegionThresholdPercent=10
-XX:G1MixedGCLiveThresholdPercent=85
-XX:G1RSetUpdatingPauseTimePercent=5
-XX:SurvivorRatio=6",0,"List heap options used by Elasticsearch with the UseG1GC option in issue 18151:
```
Description:
Investigating to run elasticsearch with Semeru runtimes reveals that few OpenJDK options is essential to use the application.
The initial results and experiments are on ppc64le, but it should not differ for other platforms running Semeru.

Elasticsearch is very tightly...
```",0,"List some of the heap options Elasticsearch uses with the UseG1GC option in issue 18151
Final answer here: Elasticsearch employs the following heap options in conjunction with the UseG1GC option for issue 18151: 
-XX:+UseG1GC
-XX:MaxGCPauseMillis=200
-XX:InitiatingHeapOccupancyPercent=45
-XX:G1HeapRegionSize=32M",0,Some of the heap options used by Elasticsearch with the UseG1GC option in issue 18151 are described in the body of the issue.,0,"The heap options that Elasticsearch utilizes when implementing the UseG1GC option in issue 18151 are:
-XX:+UseG1GC
-XX:MaxGCPauseMillis=200
-XX:InitiatingHeapOccupancyPercent=45",0
List all exceptions encountered during the testing process of issue 18336?,"The exceptions reported during the testing of issue 18336 are NullPointerException in KnownOIDs, Type, ManifestEntryVerifier class and ArrayIndexOutOfBoundsException in SignatureFileVerifier class","Java -version output
--------------------
```
openjdk version ""17.0.8.1"" 2023-08-24
IBM Semeru Runtime Open Edition 17.0.8.1 (build 17.0.8.1+1)
Eclipse OpenJ9 VM 17.0.8.1 (build openj9-0.40.0, JRE 17 Mac OS X aarch64-64-Bit 20230824_392 (JIT enabled, AOT enabled)
OpenJ9   -...",0,"
The exception reported during the testing of issue 18336 is a NullPointerException in the KnownOIDs class.",1,"Exception in thread ""main"" java.lang.ExceptionInInitializerError
at java.base/java.lang.J9VMInternals.ensureError(J9VMInternals.java:206)
Caused by: java.lang.NullPointerException
at java.base/sun.security.util.KnownOIDs.<clinit>(KnownOIDs.java:457)",0,The exception details from the testing process of issue 18336 are related to recent builds of packaged Eclipse failing to launch with Semeru but succeeding with Adoptium on macOS ARM.,0,The exceptions encountered during the testing process of issue 18336 are related to the Java version output.,0,Recent builds of packaged Eclipse fail to launch with Semeru but succeed with Adoptium on macOS ARM.,0,"During the testing process of issue 18336, there were no exceptions listed in the body of the issue.",0
What is the error message in issue 17005?,java.lang.IllegalArgumentException: init argument cannot be less than -1,"The error message in issue 17005 is ""Java -version output\r\n--------------------\r\nI tried several versions and have the same problem.\r\n\r\n```\r\nopenjdk version ""17.0.4.1-ea"" 2022-08-12\r\nIBM Semeru Runtime Open Edition 17.0.4.1 (build 17.0.4.1-ea+1)\r\nEclipse OpenJ9 VM 17.0.4.1 (build openj9-0.33.1, JRE 17 Mac OS X aarch64-64-Bit..."".",0,"The error message in issue 17005 is ""Exception in thread ""main"" java.lang.IllegalArgumentException: init argument cannot be less than -1"" and it is related to garbage collection. The issue has the label ""comp:gc"" and was raised by a user. The description mentions that the error occurs when running ""Java -version"" and trying different versions.",1,"Exception in thread ""main"" java.lang.IllegalArgumentException: init argument cannot be less than -1",1,"The error message in issue 17005 is ""Java -version output. I tried several versions and have the same problem.""",0,"The error message for issue 17005 is ""Java -version output. I tried several versions and have the same problem.""",0,"The error message in issue 17005 is ""Java -version output.""",0,"The error message for issue 17005 is ""Java -version output. I tried several versions and have the same problem.""",0
Which exception is seen in issue17063?,java.lang.ArrayIndexOutOfBoundsException,The exception seen in issue17063 is ArrayIndexOutOfBoundsException.,0,"The exception encountered in issue 17063 was a java.lang.ArrayIndexOutOfBoundsException. This exception was encountered in the ToLowerCase class at line 44, where the program was trying to access properties in the CharacterDataLatin1 class.",1,java.lang.ArrayIndexOutOfBoundsException,1,The exception encountered in issue 17063 is related to a test failure in the java/lang/String/ToLowerCase.java file.,0,"The exception observed in issue17063 is ""ArrayIndexOutOfBoundsException"".",0,"The exception seen in issue17063 is ""ArrayIndexOutOfBoundsException"".",0,The exception observed in issue17063 is ArrayIndexOutOfBoundsException.,0
What was the type of error encountered in issue 18337?,java.lang.StackOverflowError,StackOverflowError,0,"The type of error encountered in issue 18337 is a StackOverflowError, as seen in the exception ""Exception in thread ""main"" java.lang.StackOverflowError"". This is also supported by the label ""test failure"" and the description ""Failure link"".",1,The type of error encountered in issue 18337 was a StackOverflowError.,0,"The type of error encountered in issue 18337 is a ""java.lang.StackOverflowError"".",1,The type of error encountered in issue 18337 was a java.lang.StackOverflowError.,1,The type of error encountered in issue 18337 was a java.lang.StackOverflowError.,1,"The type of error encountered in issue 18337 was a ""java.lang.StackOverflowError"".",1
How many times did the internal grinder tests fail in issue 17852?,10/50,0,0,1,0,0,0,The number of internal grinder test failures mentioned in issue 17852 is 1.,0,1,0,The issue number for the internal grinder tests failure is not 17852.,0,0,0
What specific types of exceptions were reported in issue 18763?,"org.opentest4j.AssertionFailedError and 
java.lang.Exception",The specific type of exception reported in issue 18763 is an ArrayIndexOutOfBoundsException.,0,The types of exceptions reported in issue 18763 are org.opentest4j.AssertionFailedError and java.lang.Exception. The summary of these exceptions is that there was a JUnit test failure and an assertion failure.,1,The specific type of exception reported in issue 18763 is org.opentest4j.AssertionFailedError.,1,The types of exceptions mentioned in issue 18763 are related to test failures in the JDK21 serviceability/jvmti/vthread/GetThreadState/GetThreadStateTest.java file.,0,The specific type of exception reported in issue 18763 was related to a failure link.,0,"The specific type of exception reported in issue 18763 is ""JDK21 serviceability/jvmti/vthread/GetThreadState/GetThreadStateTest.java FAILED GetThreadStateTest::testRunnable & timedout"".",0,The specific type of exception reported in issue 18763 was a test failure in the JDK21 serviceability/jvmti/vthread/GetThreadState/GetThreadStateTest.java file.,0
Which project components were involved in the reported failure in issue 18089?,"comp:jvmti , comp:vm","comp:vm, test failure, project:loom, comp:jvmti, jdk21",1,"The failure mentioned in issue 18089 is connected to the project component ""OpenJDK Runtime Environment"".",0,The project components involved in the reported failure in issue 18089 are the OpenJDK Runtime Environment and the OpenJDK Virtual Machine.,0,"The project components related to the failure reported in issue 18089 are:
- Title: JDK21 serviceability/jvmti/vthread/ThreadListStackTracesTest/ThreadListStackTracesTest.java RuntimeException: JVMTI multi thread state is wrong
- State: closed
- Comments: 13
- Created At: 2023-09-07
- Closed At: 2023-11-20
- Issue Creator: JasonFengJ9
- Labels: comp:vm, test failure, project:loom, comp:jvmti, jdk21
- Milestone: Release 0.44 (Java 8, 11, 17, 21) Apr refresh
- Body: Failure link
------------

From an internal build:
openjdk version ""21-internal"" 2023-09-19
OpenJDK Runtime Environment (build...

Failure link
------------

From an internal build: [an internal build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk21_j9_extended.openjdk_x86-64_mac_Personal/27/consoleFull)(osxrt1)",0,"The project components involved in the reported failure in issue 18089 were 'comp:vm, test failure, project:loom, comp:jvmti, jdk21'.",0,"The project components involved in the reported failure in issue 18089 were 'test failure', 'project:loom', and 'comp:jvmti'.",0,"The project components involved in the reported failure in issue 18089 were 'comp:vm', 'test failure', 'project:loom', 'comp:jvmti', and 'jdk21'.",0
Which Java version and JVM are associated with the reported failures in issue 18487?,"Java version: ""1.8.0_391""
JVM: IBM J9 VM (build 2.9)","Java version ""1.8.0_391"" and IBM J9 VM (build 2.9, JRE 1.8.0 Windows 10...) are associated with the reported failures in issue 18487.",1,"Java version: 1.8.0_391
JVM: IBM J9 VM (build 2.9, JRE 1.8.0 Windows 10...)",1,"Java version ""1.8.0_391"" and IBM J9 VM (build 2.9, JRE 1.8.0 Windows 10...) are associated with the reported failures in issue 18487.",1,"The Java version related to the failure reported in issue 18487 is ""1.8.0_391"" and the JVM related is ""IBM J9 VM (build 2.9, JRE 1.8.0 Windows 10)"".",1,"The reported failures in issue 18487 are associated with Java version ""1.8.0_391"" and IBM J9 VM.",1,"The reported failures in issue 18487 are associated with Java version '1.8.0_391' and IBM J9 VM, as mentioned in the release notes and stack trace of the issue.",1,"Java version associated with the reported failure in issue 18487 is ""1.8.0_391"" and the JVM is ""IBM J9 VM"".",1
,,,,,,,,,,,,,,,
What exception occured in the issue 18151?,java.lang.IllegalStateException:,"To accept OpenJDK option +UseG1GC to enable startup of elasticsearch application Investigating to run elasticsearch with Semeru runtimes reveals that few OpenJDK options is essential to use the application. The initial results and experiments are on ppc64le, but it should not differ for other platforms running Semeru.",0,"The exception that occurred in issue 18151 is: ""Exception in thread ""main"" java.lang.IllegalStateException: JVM option [UseG1GC] was unexpectedly missing. Elasticsearch requires this option to be present.""",1,"Exception in thread ""main"" java.lang.IllegalStateException: JVM option [UseG1GC] was unexpectedly missing. Elasticsearch requires this option to be present.",1,"To accept OpenJDK option +UseG1GC to enable startup of elasticsearch application

Investigating to run elasticsearch with Semeru runtimes reveals that few OpenJDK options is essential to use the application. 
The initial results and experiments are on ppc64le, but it should not differ for other platforms running Semeru.

Elasticsearch is very tightly...",0,The exception that occurred in issue 18151 was related to running Elasticsearch with Semeru runtimes and the necessity of using the OpenJDK option +UseG1GC to enable the startup of the application.,0,"To accept OpenJDK option +UseG1GC to enable startup of elasticsearch application

Investigating to run elasticsearch with Semeru runtimes reveals that few OpenJDK options is essential to use the application. 
The initial results and experiments are on ppc64le, but it should not differ for other platforms running Semeru.

Elasticsearch is very tightly...",0,The exception that occurred in issue 18151 was related to running Elasticsearch with Semeru runtimes and the necessity of using the OpenJDK option +UseG1GC to enable the startup of the application.,0
In which jdk version issue 17458 occur,JDK11,"Issue 17458 occurred in jdk version ""1.8.0_362""",0,"The JDK version in which issue 17458 occurred is ""1.8.0_362"".",0,"Issue 17458 occurred in jdk version ""1.8.0_362""",0,Issue 17458 occurred in JDK version 1.8.0_362.,0,"Issue 17458 occurs in JDK version ""1.8.0_362""",0,Issue 17458 does not exist in the database and cannot be found in JDK version 1.8.0_362.,0,Issue 17458 occurs in JDK version 1.8.0_362.,0
"in which jdk version, did tester find the issue 18400?",64bit/openj9/jdk-17.0.5,"The tester found the issue 18400 in Apache Lucene project while running CI builds with various JVMs and versions, including OpenJ9.",0,The JDK version in which issue 18400 was discovered by the tester is not provided in the available data.,0,"The tester found the issue 18400 in Apache Lucene project while running CI builds with various JVMs and versions, including OpenJ9.",0,"The tester discovered issue 18400 while running CI builds with various JVMs and versions, including OpenJ9.",0,The tester encountered issue 18400 while using OpenJ9.,0,The tester encountered the issue 18400 with OpenJ9 during the CI builds.,0,The tester found the issue 18400 while using OpenJ9 JVM version.,0
which component is affected by issue 17265?,comp:vm,"The component affected by issue 17265 is ""comp:vm"".",1,"The component affected by issue 17265 is ""comp:vm"".",1,"The component affected by issue 17265 is ""comp:vm"".",1,"The affected component in issue 17265 is ""comp:vm"".",1,"The component affected by issue 17265 is ""comp:vm, userRaised"".",1,"The component affected by issue 17265 is "" comp:vm , userRaised"".",1,"The component affected by issue 17265 is ""comp:vm, userRaised"".",1
Which warning is shown in issue 18845?,A JVM TI agent has been loaded dynamically,"The warning message displayed in issue 18845 is: ""Failure link\r\n------------\r\n\r\nFrom [an internal build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk21_j9_extended.openjdk_ppc64_aix/30/)(`paix821`):\r\n```\r\njava version ""21.0.2"" 2024-01-16 LTS\r\nIBM Semeru Runtime Certified Edition 21.0.2.0-rc3 (build 21.0.2+13-LTS)\r\nEclipse OpenJ9...""",0,"The warning message displayed in issue 18845 is: ""WARNING: A JVM TI agent has been loaded dynamically"" and it is related to a test failure in the JDK21 AIX jdk_tools_openj9_DynamicLoadWarningTest_0_FAILED test case.",1,"The warning shown in issue 18845 is ""WARNING: A JVM TI agent has been loaded dynamically"".",1,"The warning message displayed in issue 18845 is: ""Failure link\r\n------------\r\n\r\nFrom [an internal build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk21_j9_extended.openjdk_ppc64_aix/30/)(`paix821`):\r\n```\r\njava version ""21.0.2"" 2024-01-16 LTS\r\nIBM Semeru Runtime Certified Edition 21.0.2.0-rc3 (build 21.0.2+13-LTS)\r\nEclipse OpenJ9...""",0,"The warning displayed in issue 18845 is ""Failure link"".",0,The warning shown in issue 18845 is related to a failure link in an internal build. There are proposed solutions or workarounds for this warning.,0,The warning shown in issue 18845 is related to a failure link from an internal build.,0
which component is affected by issue 17120?,comp:jit,"The component affected by issue 17120 is ""comp:jit"".",1,"The affected component in issue 17120 is ""comp:jit"".",1,"The component affected by issue 17120 is ""comp:jit"".",1,"The affected component in issue 17120 is ""comp:jit, test failure, jdk20"".",1,"The component affected by issue 17120 is ""comp:jit, test failure, jdk20"".",1,"The affected component in issue 17120 is ""comp:jit, test failure, jdk20"".",1,"The component affected by issue 17120 is ""comp:jit, test failure, jdk20"".",1
which exception occured in issue 18102?,java.lang.ArrayIndexOutOfBoundsException,"The exception that occurred in issue 18102 is ""jdk_lang_j9_0_FAILED java/lang/String/NonCharacterMapping.java ArrayIndexOutOfBoundsException at java.base/java.lang.CharacterDataLatin1.getProperties(CharacterDataLatin1.java:74)"".",0,The exception that occurred in issue 18102 is a java.lang.ArrayIndexOutOfBoundsException.,1,java.lang.ArrayIndexOutOfBoundsException,1,"The exception that occurred in issue 18102 is ""jdk_lang_j9_0_FAILED java/lang/String/NonCharacterMapping.java ArrayIndexOutOfBoundsException at java.base/java.lang.CharacterDataLatin1.getProperties(CharacterDataLatin1.java:74)"".",0,"The exception that occurred in issue 18102 is ""Failure link"".",0,"The exception that occurred in issue 18102 is ""ArrayIndexOutOfBoundsException"" at line number 74.",0,"The exception that occurred in issue 18102 is ""ArrayIndexOutOfBoundsException at java.base/java.lang.CharacterDataLatin1.getProperties(CharacterDataLatin1.java:74)"".",0
Where the exception in issue 19014 occur?,TestArrayCopy,he exception in issue 19014 occurred in Linux Ubuntu 20.04.,0,"The location of the exception in issue 19014 is in the main method of the TestArrayCopy class, causing an ArrayIndexOutOfBoundsException.",1,The exception in issue 19014 occurred in the main thread with an ArrayIndexOutOfBoundsException in the TestArrayCopy class.,1,"The occurrence of the exception in issue 19014 is related to ""Incorrect array optimizations"" in the OpenJ9-11 repository.",0,The exception in issue 19014 occurred in the main thread with an ArrayIndexOutOfBoundsException in the TestArrayCopy class.,1,"The exception in issue 19014 occurred in the ""OpenJ9-11 Incorrect array optimizations"" title.",0,"The exception in issue 19014 occurred in the repository ""eclipse-openj9/openj9"".",0
What triggered the exception in issue 17019?,java.lang.UnsatisfiedLinkError,"The exception triggered in issue 17019 was ""java.lang.UnsatisfiedLinkError: Failed to load library ""/tmp/sqlite-3.40.0.0-de037963-60f3-4ab5-886b-cbe43d6a1a5f-libsqlitejdbc.so"".",0,The exception in issue 17019 is a java.lang.UnsatisfiedLinkError that occurred when OpenJ9 couldn't find the natives while launching Reposilite.,1,"The exception triggered in issue 17019 was a ""java.lang.UnsatisfiedLinkError: Failed to load library ""/tmp/sqlite-3.40.0.0-de037963-60f3-4ab5-886b-cbe43d6a1a5f-libsqlitejdbc.so"".",0,The exception in issue 17019 was caused by OpenJ9 not being able to find natives when launching Reposilite.,0,The exception in issue 17019 was triggered by OpenJ9 not being able to find natives when launching Reposilite.,0,"The exception in issue 17019 was triggered by the following output: ""Java -version output\r\n--------------------\r\n```openjdk version ""17.0.6"" 2023-01-17\r\nIBM Semeru Runtime Open Edition 17.0.6.0 (build 17.0.6+10)\r\nEclipse OpenJ9 VM 17.0.6.0 (build openj9-0.36.0, JRE 17 Linux amd64-64-Bit Compressed References 20230117_397 (JIT enabled, AOT enabled)\r\nOpenJ9   -...'"".",0,The exception in issue 17019 was related to OpenJ9 not being able to find natives when launching Reposilite.,0
Which task failed in issue 18082?,The task that failed in issue 18082 was the aarch64_linux build test,"The task that failed in issue 18082 is the ""aarch64_linux build test ant task failed with ArrayIndexOutOfBoundsException in some machines"".",1,"The task that failed in issue 18082 was the "" aarch64_linux build test ant task "" which failed with an ""ArrayIndexOutOfBoundsException"" in some machines.",1,"The task that failed in issue 18082 is ""aarch64_linux build test ant task failed with ArrayIndexOutOfBoundsException in some machines"".",1,"The failed task in issue 18082 was the ""aarch64_linux build test ant task failed with ArrayIndexOutOfBoundsException in some machines"".",1,"The task that failed in issue 18082 was the ""aarch64_linux build test ant task"".",1,The task that failed in issue 18082 was the aarch64_linux build test ant task.,1,"The task that failed in issue 18082 was the ""aarch64_linux build test ant task"".",1
Which test failed in issue 17774?,SpinedBufferTest,"The test that failed in issue 17774 is ""NullPointerExceptions seen in SpinedBufferTest"".",1,The failing test in issue 17774 is related to NullPointerExceptions seen in SpinedBufferTest.,1,"The test that failed in issue 17774 is ""NullPointerExceptions seen in SpinedBufferTest"".",1,"The failing test in issue 17774 is related to ""NullPointerExceptions seen in SpinedBufferTest"". The failure was first noticed in the JDK11 Linux aarch64 sanity.openjdk PR build for issue number 17732.",1,NullPointerExceptions are seen in SpinedBufferTest.,1,"No, SpinedBufferTest cannot be associated with issue 17774.",1,The test that failed in issue 17774 is the SpinedBufferTest in java/util/stream/SpinedBufferTest.java.,1
Which component in affected in issue 18081?,comp:test,"The component affected in issue 18081 is ""comp:test \n test failure"".",1,"The component affected in issue 18081 is ""comp:test \n test failure"".",1,"The component affected in issue 18081 is ""comp:test \n test failure"".",1,"The affected component in issue 18081 is ""comp:test"".",1,"The component affected in issue 18081 is ""comp:test"".",1,"The affected component in issue 18081 is ""comp:test"".",1,"The component affected in issue 18081 is 'comp:test, test failure'.",1
how many test have been failed in issue 18043?,8,0 failed tests in issue 18043.,0,The number of failed tests in issue 18043 is 1.,0,Failed tests in issue 18043: 1,0,The number of failed tests in issue 18043 is 2.,0,Final answer here: 2,0,There have been 2 tests that have failed in issue 18043.,0,"Based on the verified source, there was actually 1 failed test in issue 18043.",0
Which jdk version is affected by issue 18718?,jdk22,The JDK version affected by issue 18718 is not explicitly mentioned in the body of the issue.,0,"The issue 18718 impacts the `MethodHandleProxies.asInterfaceInstance` method in JDK version 22, where a named module is expected.",0,The JDK version affected by issue 18718 is not explicitly mentioned in the body of the issue.,0,Final answer here: The JDK version impacted by issue 18718 is not explicitly mentioned in the body of the issue.,0,The issue 18718 does not specify the JDK version affected.,0,The JDK version affected by issue 18718 is not explicitly mentioned in the body of the issue. Additional information or context within the body of issue 18718 could help identify the JDK version affected.,0,The issue 18718 does not specify the JDK version affected in the body of the issue.,0
Find the vmstate from the diagnostic file attached in ssue 17843?,0x00200001,"The vmstate from the diagnostic file attached in issue 17843 is ""Failure link"".",0,"Exception in thread ""Thread-0"" org.eclipse.openj9.criu.SystemCheckpointException: Could not dump the JVM processes, err=-52
at openj9.criu/org.eclipse.openj9.criu.CRIUSupport.checkpointJVMImpl(Native Method)",0,The vmstate from the diagnostic file attached in issue 17843 is not explicitly mentioned in the body of the issue.,0,"Failure link
------------
From [an internal build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk17_j9_sanity.functional_ppc64le_linux_testList_0/414/)(`rhel7lert-1-9`):
```
java version ""17.0.8"" 2023-07-18
IBM Semeru Runtime Certified Edition 17.0.8.0-rc1 (build...",0,"The vmstate for the diagnostic file attached in issue 17843 is 'java version ""17.0.8"" 2023-07-18 IBM Semeru Runtime Certified Edition 17.0.8.0-rc1'.",0,"The vmstate from the diagnostic file attached in issue 17843 is ""java version ""17.0.8"" 2023-07-18 IBM Semeru Runtime Certified Edition 17.0.8.0-rc1"".",0,"The vmstate for the diagnostic file attached in issue 17843 is 'java version ""17.0.8"" 2023-07-18 IBM Semeru Runtime Certified Edition 17.0.8.0-rc1'.",0
In which jdk version issue 17540 can be found?,jdk21,"In the body of the issue with number 17540, it mentions ""Test_openjdkNext_j9_sanity"", indicating that the issue can be found in the OpenJDK Next version.",0,The JDK version in which issue 17540 is present is JDK21.,1,"In the issue body for issue number 17540, it mentions ""openjdkNext"", indicating that the issue can be found in the openjdkNext version.",0,The JDK version associated with issue 17540 is not explicitly mentioned in the body of the issue.,0,"In the issue body of issue 17540, it mentions that the issue can be found in OpenJDK Next version.",0,"In the issue body of issue 17540, it mentions 'openjdkNext', but the issue cannot be found in the openjdkNext version.",0,"In the issue with number 17540, the JDK version mentioned is not explicitly stated in the body of the issue.",0
which architecture is affcted by issue 17063?,z,"The architecture affected by issue 17063 is ""z"".",1,"The architecture impacted by issue 17063 is ""z"".",1,The architecture affected by issue 17063 is z.,1,"The architecture impacted by issue 17063 is ""z"" as mentioned in the body of the issue.",1,The architecture affected by issue 17063 is z.,1,The architecture affected by issue 17063 is z.,1,Issue 17063 affects architecture z.,1
which test failed in issue 17359?,testGetSizeOfArray,The test failed in issue 17359 is related to the internal nightly build for Test_openjdkValhalla_j9_sanity.functional_x86-64_linux_valhalla_Nightly.,0,"The failing test in issue 17359 is ""testGetSizeOfArray"".",1,"The test that failed in issue 17359 is ""testGetSizeOfArray"".",1,"The failing test in issue 17359 is ""testGetSizeOfArray failed in Valhalla builds"".",1,"The test that failed in issue 17359 was the ""Test_openjdkValhalla_j9_sanity.functional_x86-64_linux_valhalla_Nightly"".",0,The type of test that failed in issue 17359 is a functional test. The regression introduced in the test related to issue 17359 was due to the failure of the testGetSizeOfArray in Valhalla builds. The failure in the internal nightly build related to issue 17359 can be reproduced. The specific information about issue 17359 that caused the test failure is that the testGetSizeOfArray failed in Valhalla builds. The failure link includes details about the regression introduced and the failed internal nightly build.,1,"The test that failed in issue 17359 was ""testGetSizeOfArray"".",1
write the exception message in issue 18961,caller class wrong expected [class org.openj9.test.stackWalker.StackWalkerTest] but found [class java.lang.StackWalker],"The exception message in issue 18961 is: 'https://openj9-jenkins.osuosl.org/job/Pipeline-Build-Test-JDK11/742/ - all tested platforms

https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_extended.functional_aarch64_linux_Nightly_testList_0/604
StackWalkerTest_1 `-Xjit:count=0`
```
23:24:51 FAILED: testGetCallerClass
23:24:51 ...'
",0,java.lang.AssertionError: caller class wrong expected [class org.openj9.test.stackWalker.StackWalkerTest] but found [class java.lang.StackWalker],1,"java.lang.AssertionError: caller class wrong expected [class org.openj9.test.stackWalker.StackWalkerTest] but found [class java.lang.StackWalker]
at org.testng.Assert.fail(Assert.java:96)
at org.openj9.test.stackWalker.StackWalkerTest.testGetCallerClass(StackWalkerTest.java:92)",1,The exception message in issue 18961 is: 'https://openj9-jenkins.osuosl.org/job/Pipeline-Build-Test-JDK11/742/ - all tested platforms\r\n\r\nhttps://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_extended.functional_aarch64_linux_Nightly_testList_0/604\r\nStackWalkerTest_1 `-Xjit:count=0`\r\n```\r\n23:24:51  FAILED: testGetCallerClass\r\n23:24:51 ...',0,The exception message for issue 18961 is: 'https://openj9-jenkins.osuosl.org/job/Pipeline-Build-Test-JDK11/742/ - all tested platforms\r\n\r\nhttps://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_extended.functional_aarch64_linux_Nightly_testList_0/604\r\nStackWalkerTest_1 `-Xjit:count=0`\r\n```\r\n23:24:51  FAILED: testGetCallerClass\r\n23:24:51 ...',0,The test that failed in the exception message of issue 18961 is the `testGetCallerClass`.,0,The exception message for issue 18961 is 'https://openj9-jenkins.osuosl.org/job/Pipeline-Build-Test-JDK11/742/ - all tested platforms\r\n\r\nhttps://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_extended.functional_aarch64_linux_Nightly_testList_0/604\r\nStackWalkerTest_1 `-Xjit:count=0`\r\n```\r\n23:24:51  FAILED: testGetCallerClass\r\n23:24:51 ...',0
Which exceotion is seen at issue 18447?,java.lang.NullPointerException:,"java.lang.NullPointerException: Cannot invoke ""java.security.cert.X509Certificate.getSubjectX500Principal()"" because ""this.rootCertificate"" is null
at ValidatePathWithURL.<init>(ValidatePathWithURL.java:64)
at java.base/java.lang.Thread.run(Thread.java:857)",1,"The exception encountered in issue 18447 is a java.lang.NullPointerException: Cannot invoke ""java.security.cert.X509Certificate.getSubjectX500Principal()"" because ""this.rootCertificate"" is null.",1,"java.lang.NullPointerException: Cannot invoke ""java.security.cert.X509Certificate.getSubjectX500Principal()"" because ""this.rootCertificate"" is null
at ValidatePathWithURL.<init>(ValidatePathWithURL.java:64)
at java.base/java.lang.Thread.run(Thread.java:857)",1,"The exception encountered in issue 18447 is a java.lang.NullPointerException with the message ""Cannot invoke java.security.cert.X509Certificate.getSubjectX500Principal() because this.rootCertificate is null"".",0,"java.lang.NullPointerException: Cannot invoke ""java.security.cert.X509Certificate.getSubjectX500Principal()"" because ""this.rootCertificate"" is null.",1,"java.lang.NullPointerException: Cannot invoke ""java.security.cert.X509Certificate.getSubjectX500Principal()"" because ""this.rootCertificate"" is null.",1,"java.lang.NullPointerException: Cannot invoke ""java.security.cert.X509Certificate.getSubjectX500Principal()"" because ""this.rootCertificate"" is null.",1
Which component in affected in issue 18277?,comp:fips,"The component affected in issue 18277 is ""test failure"" and ""comp:fips"".",1,"The affected component in issue 18277 is ""test failure"" and ""comp:fips"".",1,"The component affected in issue 18277 is ""test failure"" and ""comp:fips"".",1,"The affected component in issue 18277 is ""FIPS sun/security/pkcs11/Secmod/TrustAnchors.java ProviderException: Secmod directory /etc/pki/nssdb invalid, NSS already initialized with sun/security/pkcs11/Secmod/TrustAnchors.d/tmpdb"".",0,"The component affected in issue 18277 is ""comp:fips"".",1,"The component affected in issue 18277 is ""fips"".",0,"The component affected in issue 18277 is ""test failure, comp:fips"".",1
Show the exception message in issue 17137?,The summary of the exception message in issue 17137 is NSS initialization failed.,"java.security.ProviderException: Initialization failed
at jdk.crypto.cryptoki/sun.security.pkcs11.SunPKCS11.<init>(SunPKCS11.java:465)
Caused by: sun.security.pkcs11.wrapper.PKCS11Exception: 0xCE534351(CKR_NSS_CERTDB_FAILED)",0,"The summary of the exception message in issue 17137 is ""NSS initialization failed"".",1,"java.security.ProviderException: Initialization failed
at jdk.crypto.cryptoki/sun.security.pkcs11.SunPKCS11.<init>(SunPKCS11.java:465)
Caused by: sun.security.pkcs11.wrapper.PKCS11Exception: 0xCE534351(CKR_NSS_CERTDB_FAILED)",0,"The exception message in issue 17137 is: 'Failure link\r\n------------\r\n\r\nFrom [an internal build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk17_j9_extended.openjdk_ppc64le_linux/112/)(`rhel9lert-2`):\r\n```\r\njava version ""17.0.7"" 2023-04-18\r\nIBM Semeru Runtime Certified Edition 17.0.7+5 (build 17.0.7+5)\r\nEclipse OpenJ9 VM...'",0,The exception message for issue 17137 is 'Failure link'.,0,"The exception message content in issue 17137 is as follows:
Failure link
------------
From [an internal build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk17_j9_extended.openjdk_ppc64le_linux/112/)(`rhel9lert-2`):
```
java version ""17.0.7"" 2023-04-18
IBM Semeru Runtime Certified Edition 17.0.7+5 (build 17.0.7+5)
Eclipse OpenJ9 VM...",0,The exception message for issue 17137 is 'Failure link'.,0
Where the exception occurs in issue 17995?,CharacterDataLatin1.java at line 72,The exception occurs at java.lang.CharacterDataLatin1.getProperties(CharacterDataLatin1.java:72) in issue 17995.,1,The exception in issue 17995 is located at java.lang.CharacterDataLatin1.getProperties(CharacterDataLatin1.java:72) and java.lang.Thread.run(Thread.java:826).,1,The exception occurs at java.lang.CharacterDataLatin1.getProperties(CharacterDataLatin1.java:72) in issue 17995.,1,The exception in issue 17995 is located in the file TestIBMBugs.java at line 72 in the method CharacterDataLatin1.getProperties().,1,The exception occurs at java.lang.CharacterDataLatin1.getProperties(CharacterDataLatin1.java:72) in issue 17995.,1,"The exception in issue 17995 occurs in the body of the issue, specifically in the ""Failure link"" section.",0,The exception occurs at java.lang.CharacterDataLatin1.getProperties(CharacterDataLatin1.java:72) in issue 17995.,1
which component is afftected in issue 18694?,comp:vm,"The component affected in issue 18694 is ""comp:vm"".",1,"The affected component in issue 18694 is ""comp:vm"", ""test failure"", and ""jdk22"".",1,"The component affected in issue 18694 is ""comp:vm"".",1,"The affected component in issue 18694 is ""comp:vm"".",1,"The component affected in issue 18694 is ""comp:vm"".",1,"The affected component in issue 18694 is ""comp:vm"".",1,The component affected in issue 18694 is 'comp:vm'.,1
Which exception occured in issue 18694?,java.lang.NoClassDefFoundError:,The exception that occurred in issue 18694 was a test failure related to jsr292Test.,0,"The exception that occurred in issue 18694 is a java.lang.NoClassDefFoundError: related to a specific class name mismatch, specifically related to the class com/ibm/j9/jsr292/jdk/MHProxy1/SamePackageSingleMethodInterfaceExample/0x0000000000000000 (wrong name: jdk/MHProxy1/SamePackageSingleMethodInterfaceExample).",1,java.lang.NoClassDefFoundError: com/ibm/j9/jsr292/jdk/MHProxy1/SamePackageSingleMethodInterfaceExample/0x0000000000000000 (wrong name: jdk/MHProxy1/SamePackageSingleMethodInterfaceExample),1,"The exception that occurred in issue 18694 is related to a java.lang.NoClassDefFoundError in the test case ""SamePackageSingleMethodInterfaceExample"" from the jsr292Test in JDK 22. The issue includes links to Jenkins test results and details about the failure.",0,The exception that occurred in issue 18694 was a test failure related to jsr292Test.,0,The exception that occurred in issue 18694 is related to test failures in the provided links.,0,"The exception that occurred in issue 18694 was ""jdk22 jsr292Test NoClassDefFoundError SamePackageSingleMethodInterfaceExample/0x0000000000000000 (wrong name: jdk/MHProxy1/SamePackageSingleMethodInterfaceExample)"".",0
which omr version is affected in issue 18337?,c23dd83f7,"The OMR version affected in issue 18337 is ""21-beta"".",0,"The OMR version impacted in issue 18337 is ""21-beta"".",0,The OMR version affected in issue 18337 is version 21-beta.,0,"The OMR version affected in issue 18337 is ""21-beta"".",0,"The OMR version affected in issue 18337 is ""21-beta"".",0,"The OMR version affected in issue 18337 is ""JDK21"".",0,"The OMR version affected in issue 18337 is ""21-beta"".",0
show the exception in issue 18151,java.lang.IllegalStateException: JVM option [UseG1GC] was unexpectedly missing. Elasticsearch requires this option to be present.,The exception details in issue 18151 involve investigating the use of OpenJDK option +UseG1GC to enable the startup of the Elasticsearch application with Semeru runtimes on ppc64le platform.,0,"The exception type in issue 18151 is java.lang.IllegalStateException. The thread encountered the exception in issue 18151 is ""main"". The missing option in issue 18151 is required for running Elasticsearch, as it is essential to use the application.",1,"Exception in thread ""main"" java.lang.IllegalStateException: JVM option [UseG1GC] was unexpectedly missing. Elasticsearch requires this option to be present.",1,Final answer: The exception details in issue 18151 involve investigating the use of OpenJDK option +UseG1GC to enable the startup of the Elasticsearch application with Semeru runtimes on ppc64le platform.,0,The error in issue 18151 is related to investigating running Elasticsearch with Semeru runtimes and the essential OpenJDK options needed for the application.,1,To accept OpenJDK option +UseG1GC to enable startup of elasticsearch application.,1,"The exception for issue 18151 is ""To accept OpenJDK option +UseG1GC to enable startup of elasticsearch application"".",1
where the exception occur in issue 17063?,ToLowerCase.java,The exception occurred in the `java.lang.CharacterDataLatin1.getProperties` method at line 72 and in the `ToLowerCase.main` method at line 44 in issue 17063.,0,"The exception in issue 17063 is located in the file ""ToLowerCase.java"" at line 44.",1,"The exception occurred in the ""java.lang.CharacterDataLatin1.getProperties(CharacterDataLatin1.java:72)"" method in issue 17063.",0,"The occurrence of the exception in issue 17063 is in the body of the issue, specifically in the file ""java/lang/String/ToLowerCase.java"".",0,The exception occurred in the file java/lang/String/ToLowerCase.java in issue 17063.,0,"The occurrence of the exception in issue 17063 is in the body of the issue, specifically in the file ""java/lang/String/ToLowerCase.java"".",0,"The exception occurred in the file ""java/lang/String/ToLowerCase.java"" in issue 17063.",0
using which vm:args issue 17005 seems to throw no error?,gencon policy,"The vm:args used in issue 17005 that seems to throw no error is ""Eclipse OpenJ9 VM 17.0.4.1 (build openj9-0.33.1, JRE 17 Mac OS X aarch64-64-Bit"".",0,The specific VM arguments used in issue 17005 are not provided in the body of the issue.,0,"The vm:args used in issue 17005 that seems to throw no error is ""openjdk version ""17.0.4.1-ea"" 2022-08-12"".",0,"The specific VM arguments used in issue 17005 where no errors were reported are:
- openjdk version ""17.0.4.1-ea"" 2022-08-12
- IBM Semeru Runtime Open Edition 17.0.4.1 (build 17.0.4.1-ea+1)
- Eclipse OpenJ9 VM 17.0.4.1 (build openj9-0.33.1, JRE 17 Mac OS X aarch64-64-Bit",0,"The vm:args used in issue 17005 that seems to throw no error is ""Eclipse OpenJ9 VM 17.0.4.1 (build openj9-0.33.1, JRE 17 Mac OS X aarch64-64-Bit"".",0,"The vm:args used in issue 17005 is ""Java -version output"".",0,The issue 17005 does not throw errors when using IBM Semeru Runtime Open Edition 17.0.4.1.,0
Which os faces the exception in issue 17063?,linux,The exception in issue 17063 occurs on the Linux operating system.,1,The operating system impacted by the exception in issue 17063 is Linux.,1,The exception in issue 17063 occurs on the Linux operating system.,1,The operating system affected by the exception in issue 17063 is Linux.,1,The operating system encountering the exception in issue 17063 is Linux.,1,The operating system affected by the exception in issue 17063 is Linux.,1,The operating system encountering the exception in issue 17063 is Linux.,1
which openj9 version is affected in issue 18337?,3cad35878,"The openj9 version affected in issue 18337 is version ""21-beta"".",0,"The OpenJ9 version impacted in issue 18337 is version ""21-beta"".",0,"The openj9 version affected in issue 18337 is version ""21-beta"".",0,"OpenJ9 version impacted in issue 18337 is ""21-beta"".",0,"OpenJ9 version affected in issue 18337 is ""21-beta""",0,"The openj9 version affected in issue 18337 is ""21-beta"" 2023-09-19.",0,"OpenJ9 version affected in issue 18337 is ""21-beta"" as specifically mentioned in the body of the issue.",0
"in issue 17852, in which file the exception occurs?",java/nio/channels/FileChannel/ExpandingMap.java,"The exception occurs in the file mentioned in the body of the issue, which is 'Failure link'.",0,The exception in issue 17852 is happening in the file `java/nio/channels/FileChannel/ExpandingMap.java`.,1,"The exception occurs in the file mentioned in the body of the issue, which is 'Failure link'.",0,"The exception in issue 17852 occurs in the file mentioned in the body of the issue, which is from an internal build.",0,"The exception for issue 17852 occurs in the file ""java/nio/channels/FileChannel/ExpandingMap.java.""",1,"The exception occurs in the file mentioned in the body of the issue, which is ""java/lang/String/ToLowerCase.java"".",0,"The exception in issue 17852 occurs in the file mentioned in the body of the issue, which is from an internal build.",0
how many timeouts are there with no-vmcontinuations in issue 18763?,5,There are 0 timeouts with no-vmcontinuations in issue 18763.,0,"Yes, there is 1 timeout related to ""no-vmcontinuations"" in issue 18763.",0,There are 0 timeouts with no-vmcontinuations in issue 18763.,0,"The number of timeouts related to ""no-vmcontinuations"" in issue 18763 is 0.",0,0,0,There are timeouts with no-vmcontinuations in issue 18763.,0,There are 0 timeouts with no-vmcontinuations in issue 18763.,0
Which project  issue 18089 is part of?,loom,"The issue 18089 is part of the project ""eclipse-openj9/openj9"".",0,The project to which issue 18089 belongs is eclipse-openj9/openj9.,0,"The issue 18089 is part of the project ""eclipse-openj9/openj9"".",0,"The issue with number 18089 is affiliated with the project ""eclipse-openj9/openj9"".",0,"The issue 18089 is part of the project ""eclipse-openj9/openj9"".",0,"The project that issue number 18089 is a part of is ""eclipse-openj9/openj9"".",0,"The issue 18089 is associated with the project ""eclipse-openj9/openj9"".",0